SPECS/lua/lua.spec

```diff
@@ -8,30 +8,47 @@
 
 Name:           lua
 Version:        %{major_version}.5
-Release:        9%{?dist}
+Release:        11%{?dist}
 Summary:        Powerful light-weight programming language
 License:        MIT
 URL:            https://www.lua.org/
 Group:          Development/Tools
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-Source0:        http://www.lua.org/ftp/lua-%{version}.tar.gz
+Source0:        https://www.lua.org/ftp/%{name}-%{version}.tar.gz
 # copied from doc/readme.html on 2014-07-18
 Source1:        mit.txt
 Source2:        http://www.lua.org/tests/lua-%{test_version}-tests.tar.gz
 # multilib
 Source3:        luaconf.h
 # rpm-macro
 Source1000:     macros.lua
+
 Patch0:         %{name}-5.3.0-autotoolize.patch
 Patch1:         %{name}-5.3.0-idsize.patch
 Patch2:         %{name}-5.2.2-configure-linux.patch
 Patch3:         %{name}-5.3.0-configure-compat-module.patch
 # Fixes CVE-2019-6706
 Patch4:         CVE-2019-6706-use-after-free-lua_upvaluejoin.patch
+Patch5:         lua-5.3.4-shared_library-1.patch
+# CVE-2020-15888 patch taken from Open Embedded's Lua meta layer https://github.com/openembedded/meta-openembedded/blob/master/meta-oe/recipes-devtools/lua/lua/CVE-2020-15888.patch
+# NOTE: Upstream patches needed if updating to 5.4:
+#   - eb41999461b6f428186c55abd95f4ce1a76217d5
+#   - 6298903e35217ab69c279056f925fb72900ce0b7
+Patch6:         CVE-2020-15888.patch
+# CVE-2020-15889 is in the Lua generational garbage collection code, which is new to 5.4.0. 5.3.5 is not affected.
+# NOTE: Patches needed if updating to 5.4:
+#   - 127e7a6c8942b362aa3c6627f44d660a4fb75312
+Patch7:         CVE-2020-15889.nopatch
+# CVE-2020-24342 appears to not affect 5.3.5 (no repro of exploit)
+# NOTE: Patches needed if updating to 5.4:
+#   - 34affe7a63fc5d842580a9f23616d057e17dfe27
+Patch8:         CVE-2020-24342.nopatch
+# From http://lua.2524044.n2.nabble.com/CVE-2019-6706-use-after-free-in-lua-upvaluejoin-function-tt7685575.html
 
 BuildRequires:  automake autoconf libtool readline-devel ncurses-devel
 Requires:       lua-libs = %{version}-%{release}
+Requires:       readline
```

Comment:
Is this needed? If there's not a known reason for `dev`'s copy of Lua not needing `readline` I think this should be omitted.

Reply:
I don't think so. It was in the 1.0 branch so was merged in by default. will remove.